python/GafferSceneTest/OpenGLRenderTest.py

```diff
@@ -108,7 +109,7 @@ def test( self ) :
 	def testOutputDirectoryCreation( self ) :
 
 		s = Gaffer.ScriptNode()
-		s["variables"].addChild( Gaffer.NameValuePlug( "renderDirectory", self.temporaryDirectory() + "/openGLRenderTest" ) )
+		s["variables"].addChild( Gaffer.NameValuePlug( "renderDirectory", ( self.temporaryDirectory() / "openGLRenderTest" ).as_posix() ) )
```

Comment:
I'd like to understand more here. Is the `as_posix()` necessary because we use `/` in `$renderDirectory/test.####.exr`, and while Windows will accept paths using `/`, it won't accept a mix of `\` and `/`?

Reply:
In this case, it is needed because the string substitution will interpret the path to look like it has escape characters instead of path separators.

Windows will accept `/` and `\` mixed together, but we'd have to be careful to make sure the `\` are escaped properly before handing it to Windows.

There's another case in `OpenGLRenderTest` where I have an `as_posix()` sent to an `IECore.Output`, which doesn't sound like it would need `as_posix()` since it's going to Cortex. But in that case, it's final destination is a `Gaffer.Outputs` node, which brings us back to the string substitution problem we have here.

Reply:
Got it - thanks for the explanation.